eglib/src/gpath.c

```diff
@@ -250,11 +250,13 @@ g_find_program_in_path (const gchar *program)
 		
 		x = NULL;
 		probe_path = g_build_path (G_DIR_SEPARATOR_S, l, program, NULL);
+#ifdef X_OK
 		if (access (probe_path, X_OK) == 0){ /* FIXME: on windows this is just a read permissions test */
```

Comment:
Checking for `X_OK` seems odd. Do you really have a platform which defines access() but not the `X_OK` symbol? Would it make more sense to add access to configure.ac and use HAVE_ACCESS?